packages/go_router_builder/example/lib/main.dart

```diff
@@ -226,7 +226,7 @@ class PersonScreen extends StatelessWidget {
                     person.id,
                     entry.key,
                     $extra: ++_extraClickCount,
-                  ).go(context),
```

Comment:
instead of changing go into push, can you add a new button for push in parallel to this original button?

Reply:
might be good to also add a test in `example/test` to make sure the push works correctly.